src/zzz_od/application/notorious_hunt/notorious_hunt_config.py

```
@@ -47,47 +47,41 @@ def __init__(self, instance_idx: int, group_id: str):
-        if len(self.plan_list) < len(default_list):
-            for plan in default_list:
-                if plan.mission_type_name not in existed_missions:
-                    self.plan_list.append(plan)
+        for plan_item in self.data.get('plan_list', []):
+            self.plan_list.append(ChargePlanItem(**plan_item))
+
+        # 旧配置迁移，2026-09-21 可删除
+        self._migrate_legacy_config()
+
+    def _migrate_legacy_config(self) -> None:
+        """迁移旧配置，2026-09-21 可删除。"""
```

Ruff (0.15.15)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)